### PR TITLE
Jenkinsfile update

### PR DESCRIPTION
**Pull request by dasumner**

### CODE REVIEW

File: getconfig/getconfig.sh [hunk 1]
```diff
@@ -13,26 +13,27 @@ fi
 
 echo "Updating PHP configuration"
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
-scp "$DIR"/getconfig.php "${HOST}":/tmp/
+scp -i /Users/davidsumner/.ssh/id_rsa_4096 "$DIR"/getconfig.php ridata.prod@"${HOST}":/tmp/
+echo "Ping"
 for PHP in "${SUPPORTED_PHP[@]}"; do
     echo "Getting config for $HOST -> $PHP"
-    ssh "$HOST" "/usr/local/php${PHP}/bin/php" /tmp/getconfig.php acquia > php/"${PHP}"-config
+    ssh -i /Users/davidsumner/.ssh/id_rsa_4096 -l ridata.prod "$HOST" "/usr/local/php${PHP}/bin/php" /tmp/getconfig.php acquia > php/"${PHP}"-config
```
Comment (grugnog) [inline getconfig/getconfig.sh:20]: Looks like some local paths in here - perhaps add an env var or arg to set these locally when needed (could potentially use .ssh/config for some of this?)

File: php/Dockerfile [hunk 2]
```diff
@@ -34,12 +34,14 @@ ENV BUILDDEPS autoconf \
...
         libsasl2-dev \
+        libsodium-dev \
         libsqlite3-dev \
         libssh2-1-dev \
         libssl-dev \
         libsslcommon2-dev \
         libtidy-dev \
-        libxml2-dev \
+        libwebp-dev \
+	libxml2-dev \
```
Comment (grugnog) [inline php/Dockerfile:44]: Indenting?

**Review by grugnog — CHANGES_REQUESTED** [review]

File: getconfig/getconfig.sh (hunk 1 above)
Comment (dasumner) [reply getconfig/getconfig.sh:20]: The arguments weren't used in the original copies of the script.  I just added them to my local copies.  Updating the .ssh/config file is the better way to go.  I will remove them from here.

File: php/Dockerfile (hunk 2 above)
Comment (dasumner) [reply php/Dockerfile:44]: Strange.  I don't see this spacing in VS code or when I look at the file in the github editor.  I'll see if I can figure out where/how it's being added.

**Pull request closed without merging**